MergeQuery class can not be found with 2020.1

Environment:
* **org.jetbrains.grammarkit**: 2019.3
* **intellij.version**: IU-201.3803-EAP-CANDIDATE-SNAPSHOT

While parser generationI get the following exception. 
<details><summary>stacktrace</summary>
<p>

```
java.lang.NoClassDefFoundError: com/intellij/util/MergeQuery
	at com.intellij.psi.search.searches.ReferencesSearch.search(ReferencesSearch.java:173)
	at com.intellij.psi.search.searches.ReferencesSearch.search(ReferencesSearch.java:153)
	at com.intellij.psi.search.searches.ReferencesSearch.search(ReferencesSearch.java:139)
	at org.intellij.grammar.analysis.BnfFirstNextAnalyzer.calcNextInner(BnfFirstNextAnalyzer.java:152)
	at org.intellij.grammar.analysis.BnfFirstNextAnalyzer.calcFirstInner(BnfFirstNextAnalyzer.java:318)
	at org.intellij.grammar.analysis.BnfFirstNextAnalyzer.calcFirstInner(BnfFirstNextAnalyzer.java:207)
	at org.intellij.grammar.analysis.BnfFirstNextAnalyzer.calcFirst(BnfFirstNextAnalyzer.java:90)
	at org.intellij.grammar.generator.ParserGenerator.generateFirstCheck(ParserGenerator.java:1094)
	at org.intellij.grammar.generator.ParserGenerator.generateNode(ParserGenerator.java:899)
	at org.intellij.grammar.generator.ParserGenerator.generateParser(ParserGenerator.java:568)
	at org.intellij.grammar.generator.ParserGenerator.generateParser(ParserGenerator.java:521)
	at org.intellij.grammar.generator.ParserGenerator.generate(ParserGenerator.java:369)
	at org.intellij.grammar.Main.main(Main.java:85)
Caused by: java.lang.ClassNotFoundException: com.intellij.util.MergeQuery
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 13 more
```

</p>
</details>

The cause was moved `MergeQuery` class from `platform-api` module to `platform-concurrency` one. Since `gradle-grammar-kit-plugin` uses a restricted list of jars as classpath of `Grammar Kit`, it leads to NoClassDefFoundError

--- a/src/main/groovy/org/jetbrains/grammarkit/tasks/GenerateParser.groovy
+++ b/src/main/groovy/org/jetbrains/grammarkit/tasks/GenerateParser.groovy
@@ -28,7 +28,7 @@ class GenerateParser extends BaseTask {
             def requiredLibs = [
                     "jdom", "trove4j", "junit", "guava", "asm-all", "automaton", "platform-api", "platform-impl",
                     "util", "annotations", "picocontainer", "extensions", "idea", "openapi", "Grammar-Kit",
-                    "platform-util-ui",
+                    "platform-util-ui", "platform-concurrency",
                     // CLion unlike IDEA contains `MockProjectEx` in `testFramework.jar` instead of `idea.jar`
                     // so this jar should be in `requiredLibs` list to avoid `NoClassDefFoundError` exception
                     // while parser generation with CLion distribution

